Tools/ros2/ardupilot_sitl/CMakeLists.txt

```diff
@@ -99,6 +99,12 @@ install(DIRECTORY
   DESTINATION share/${PROJECT_NAME}/config/
 )
 
+# Install additional autotest model params.
+install(DIRECTORY
+  ${ARDUPILOT_ROOT}/Tools/autotest/models
```

Comment:
Do we want everything from `./Tools/autotest/models` or just the `.parm` files?

Reply:
I think it's ok to put everything in just in case people need it.